index.js

```diff
@@ -151,6 +152,15 @@ server.register(inert, () => {
 });
 
 
+if (process.env.NODE_ENV === 'production') {
```

Comment:
We could check here for another environment variable if present. Not sure if Heroku is used and what is set exactly, so let me know.